grazer/src/main/webapp/WEB-INF/views/person.jsp

```diff
@@ -26,45 +26,131 @@
 	}
 </script>
 
+<script src="<c:url value="/resources/js/cytoscape.min.js" />"></script>
+<script src="<c:url value="/resources/js/cytoscape-layouts/cytoscape-cose-bilkent.js" />"></script>
+<script>
+//# sourceURL=graph.js
+$(document).ready(function() {
+    var cy;
+    var highlightSize = "20px";
+    var nodeSize = "10px";
+    
+    $.ajax({
+        url : '<c:url value="/concept/${concept.id}/network1" />',
+        type : "GET",
+        success : function(result) {
+            if (result == null || result.length == 0) {
+                $("#spinner1").hide();
+                $("#network").append("Sorry, no network to display.")
+            } else {
+                $("#spinner1").hide();
+                data = JSON.stringify(result);
+                cy = cytoscape({
+                    container: $('#network'),
+                    elements: result,
+                    style: [ // the stylesheet for the graph
+                        {
+                            selector: 'node',
+                            style: {
+                                'background-color': '#7bafa6',
+                                'width': nodeSize,
+                                'height': nodeSize,
+                                'label': 'data(label)'
+                            }
+                        },
+                        {
+                            selector: 'edge',
+                            style: {
+                                'width': 2,
+                                'line-color': '#b0c7c3',
+                                'target-arrow-color': '#b0c7c3',
+                                'target-arrow-shape': 'triangle'
+                            }
+                        }
+                    ],
+                	    
+                    layout: {
+                        name: 'cose-bilkent',
+                        nodeDimensionsIncludeLabels: true,
+                    }
+                });
+	            
+                cy.on('tap', 'node', function(){
+                    window.location.href = this.data('id');
+                })
+	            
+                cy.ready(function() {
+                    $(".person-entry").hover(highlightPersonInGraph, removeHighlight);
+                });
+            }
+        },
+        error: function() {
+            $("#spinner1").hide();
+            $("#network").append("Sorry, could not load network.")
+        }
+    });
+	
+    function highlightPersonInGraph() {
+        var id = $(this).data("concept-id");
+        var node = cy.getElementById(id);
+        node.animate({
+            css: { 'width': highlightSize, 'height' : highlightSize},
+        });		
+    }
+    
+    function removeHighlight() {
+        var id = $(this).data("concept-id");
+        var node = cy.getElementById(id);
+        node.animate({
+            css: { 'width': nodeSize, 'height' : nodeSize}
+        });
+    }
+})
+
+</script>
```

Comment:
since this is the same as for the general overview page, this should be put into it's own js file and used in both pages.

Reply:
It is not exactly the same as the home page, there are some slight modifications that I've made for the URLs and node sizes in this file

Reply:
you can still put that in it's own js file and simply pass whatever is different